.github/workflows/upstream.yml

```diff
@@ -0,0 +1,46 @@
+name: Upstream
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.9]
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          python-version: ${{ matrix.python-version }}
+          mamba-version: '*'
+          environment-file: ci/minimum-tests.yml
+          activate-environment: xskillscore-minimum-tests
+          auto-activate-base: false
+
+      - name: Install upstream packages
+        shell: bash -l {0}
+        run: |
+          python -m pip install git+https://github.com/pydata/xarray.git
+          python -m pip install git+https://github.com/dask/dask.git
+          python -m pip install git+https://github.com/xgcm/xhistogram.git
+
+      - name: Install
+        shell: bash -l {0}
+        run: python -m pip install -e .
+
+      - name: List installed packages
+        shell: bash -l {0}
+        run: conda list
+
+      - name: Run tests
+        shell: bash -l {0}
+        run: python -m pytest --cov=xskillscore --cov-report=xml xskillscore
+
+      - name: Coverage
+        uses: codecov/codecov-action@v1
```

Comment:
Why do you want to run coverage on upstream?

Reply:
i've take this out